docs/conf.py

```diff
@@ -38,6 +38,13 @@
 # Do not warn on external images.
 suppress_warnings = ['image.nonlocal_uri']
 
+# Ignore certain warn as error messages
+nitpick_ignore = [
```

Comment:
We had to add this since the static fields in deposit API class are failing to resolve to a class-definition